homework/shelkovich_ilya/circle/src/test/java/com/tutrit/java/quickstart/service/CircleServiceTest.java

```diff
@@ -0,0 +1,29 @@
+package com.tutrit.java.quickstart.service;
+
+import com.tutrit.java.quickstart.bean.Circle;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CircleServiceTest {
+
+    Circle circle;
+    CircleService circleService;
+    double radius;
+
+    @Before
+    public void setUp(){
+        this.circle = new Circle();
+        this.circleService = new CircleService();
+        this.radius = 4.5;
```

Comment:
it's redundant to have this value in @Before. 
You could use it directly in test, that's fine

Reply:
fixed it